### PR TITLE
Related: RAIL-3531 Attribute filtering

### PR DESCRIPTION
**Pull request by plavemar**
- AttributeFilter parent child filtering docs
- AttributeFilterButton complete docs

JIRA: RAIL-3531

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/764/ : SUCCESS in 1m 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/488/ : SUCCESS in 6s

File: docs/10_vis__attribute_filter_button_component.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import { AttributeFilterButton, Model } from "@gooddata/sdk-ui-filters";
+import { newNegativeAttributeFilter } from "@gooddata/sdk-model";
+
+import "@gooddata/sdk-ui-filters/styles/css/main.css";
+
+import { Ldm } from "./ldm";
+
+export class AttributeFilteButtonrExample extends Component {
+    onApply(filter) {
+        // eslint-disable-next-line no-console
```
Comment (no23reason) [inline docs/10_vis__attribute_filter_button_component.md:32]: Please remove the eslint comment, it is not very useful in this context

File: docs/10_vis__attribute_filter_button_component.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                    onApply={this.onApply}
+                />
+            </div>
+        );
+    }
+```
+
+## Attribute parent child filtering
+
+To define parent child relationship between two filters, create two filters and hand over ```parentFilters``` and ```parentFilterOverAttribute``` properties to the one which is supposed to be child filter dependent on the another.
```
Comment (no23reason) [inline docs/10_vis__attribute_filter_button_component.md:67]: Do we have some docs on how to pick those?

File: website/i18n/en.json [hunk 3]
```diff
@@ -77,6 +77,10 @@
         "title": "Area Chart",
         "sidebar_label": "Area Chart"
       },
+      "attribute_filter_button_component": {
```
Comment (no23reason) [inline website/i18n/en.json:80]: Please put this after the original AttributeFilter component.

Comment (no23reason) [reply website/i18n/en.json:80]: Nevermind, this is the wrong file to concern ourselves with.

File: docs/10_vis__attribute_filter_button_component.md (hunk 2 above)
Comment (plavemar) [reply docs/10_vis__attribute_filter_button_component.md:67]: Added parentFilterOverAttribute docs to both, to AttributeFilter as well as to AttributeFilterButton docs.

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/765/ : SUCCESS in 2m 22s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/489/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/770/ : SUCCESS in 1m 08s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/492/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/771/ : SUCCESS in 1m 37s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/493/ : SUCCESS in 6s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by evgeniabogranskaya** [comment]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR432

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/772/ : SUCCESS in 57s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/132/ : FAILURE in 2m 32s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/34/ : SUCCESS in 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/223/ : SUCCESS in 18s

**Pull request merged**